code/game/objects/structures/tables_racks.dm

```diff
@@ -393,6 +399,7 @@
 			add_logs(user, src, "climbed onto")
 			user.Stun(2)
 			return 1
+			tableclimber = null
 	return 0

```

Comment:
You should add another tableclimber=null just above this line

Why?

Consider the case where the do_mob call fails for some unrelated reason (i.e some taser stun or some kind of collapsing disease, or the user climbing moves away and cancels their action)

We need unset the tableclimber variable as the earlier unset is inside the if block for the do_mob and hence will not be run.

Pm me on IRC if you need this explained more